schema/crdb/dbinit.sql

```diff
@@ -4815,6 +4815,9 @@ CREATE TABLE IF NOT EXISTS omicron.public.bp_sled_metadata (
     -- the sled's /64 subnet on the underlay address
     subnet INET NOT NULL,
 
+    -- the last allocated IP within `subnet` used by the blueprint
+    last_allocated_ip_subnet_offset INT4 NOT NULL,
```

Comment:
What do you think about a constraint between 0 and 65K?

Reply:
Added in [61502ee](https://github.com/oxidecomputer/omicron/pull/9550/commits/61502eeffb813b45a9a3b7b4afbda263a47d68a6)